src/DrawioClient/DrawioClientFactory.ts

```diff
@@ -231,6 +233,7 @@ export class DrawioClientFactory {
 			.replace(/\$\$literal-vsuri\$\$/g, vsuri.toString())
 			.replace("$$theme$$", JSON.stringify(config.theme))
 			.replace("$$lang$$", JSON.stringify(config.drawioLanguage))
+			.replace("$$simpleLabels$$", JSON.stringify(config.simpleLabels))
```

Comment:
This requires a change in the entry html file...
Otherwise it will only work in online mode.

Reply:
Thank you. But I see you merged. So you are adding this change in another branch?

Reply:
It works, just not in offline mode. Please create another PR for that. Thank you!